### PR TITLE
[16.0][MIG] sale_planner_calendar: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
supersedes https://github.com/OCA/sale-workflow/pull/2556

cc @Tecnativa TT44359

@CarlosRoca13 @carolinafernandez-tecnativa please review

### CODE REVIEW

File: sale_planner_calendar/static/src/js/form_controller.esm.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// /** @odoo-module **/
```
Comment (CarlosRoca13) [inline sale_planner_calendar/static/src/js/form_controller.esm.js]:
```suggestion
/** @odoo-module **/

File: sale_planner_calendar/static/src/js/list_controller.esm.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// /** @odoo-module **/
```
Comment (CarlosRoca13) [inline sale_planner_calendar/static/src/js/list_controller.esm.js]:
```suggestion
/** @odoo-module **/

Comment (pilarvargas-tecnativa) [reply sale_planner_calendar/static/src/js/list_controller.esm.js]: thanks!

**Comment by carolinafernandez-tecnativa** [comment]
I create a planner for a contact with tag Delivery

image

I create a sale order for that contact and when press button Confirm i got this error message:

image

You need to include this fix to migration:

https://github.com/OCA/sale-workflow/pull/3225

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]
Something is wrong on _Sale planner calendar summary_ form
image
vs
image

**Comment by pilarvargas-tecnativa** [comment]
> Something is wrong on _Sale planner calendar summary_ form 

Please check again

image

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]
On Configuration > Reassignment of salesperson

This buttons should be above the tree
image

And the buttons to apply the changes are missing:
image

**Comment by CarlosRoca13** [comment]
Check this steps:

1. Create a calendar recurrence for a user of 1 year
2. Go to Configuration > Reassignment of salesperson
3. Select the salesperson of the recurrence
4. Press on Fill events from filters
5. Select a period (New Start and New End)
6. Press option Assign New Salesperson
7. On the event line set the new user for the period
8. Apply Changes

An error is thrown

**Comment by carolinafernandez-tecnativa** [comment]
> I create a planner for a contact with tag Delivery
> 
> image
> 
> I create a sale order for that contact and when press button Confirm i got this error message:
> 
> image
> 
> You need to include this fix to migration:
> 
> #3225

I have tested and with recent changes this is solved.
Waiting for @CarlosRoca13 to check other things.
Thanks!

**Review by Deriman-Alonso — APPROVED** [review]
LGTM 👍, code review

**Review by CarlosRoca13 — APPROVED** [review]
Thank you so much :smile:

**Pull request merged**